src/main/resources/leonardo-example.conf

```diff
@@ -0,0 +1,113 @@
+# Example conifg file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and secruity certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  applicationName = "leonardo"
+  dataprocDefaultRegion = "us-central1"
+  leoGoogleProject = ""
+  clusterUrlBase = "https://leonardo.dsde-dev.broadinstitute.org/notebooks/"
+  jupyterServerName = "jupyter-server"
+}
+
+# Keys and certificate authorities for cluster
+clusterFiles {
+  configFolderPath = "/etc/"
+  jupyterServerCrt = "jupyter-server.crt"
+  jupyterServerKey = "jupyter-server.key"
+  jupyterRootCaPem = "rootCA.pem"
+  jupyterRootCaKey = "rootCA.key"
+}
+
+# These are the cluster defaults that have already been configured.
+# You should only set them in your config if you want a different value
+#clusterDefaults {
```

Comment:
Not all combinations here are valid, so it would be good to list our requirements.
* minimum disk size is 100 GB
* numberOfWorkers cannot be exactly 1 (0 and 2+ are both valid)
* if you have numberOfPreemptibleWorkers > 0, then numberOfWorkers must be 2 or more

See `MachineConfig` in Swagger and `MachineConfig.machineConfigSemigroup`